### PR TITLE
feat: support nestjs template

### PR DESCRIPTION
**Pull request by vmcodes**
This needs a code review and testing, but I was able to run the NestJS template Blocklet locally. It is very similar to the Express template Blocklet.

Please let me know of any next steps, or anything that I could have done myself. 

Thank you.

### CODE REVIEW

**Comment by wangshijun** [comment]
@vmcodes Thanks for the contribution!

blocklet projects created from the new template should meet the following requirements

- [ ] can be bundled with `npm run bundle`
- [ ] can be deployed with `npm run deploy`
- [ ] bump patch version with `npm run bump-version`

Since blocklet server 1.16.0, the correct steps to deploy a blocklet should be:

1. create an app container from your server dashboard
Screenshot 2023-05-23 at 05 52 47

2. get the appDid of the newly created app
image

3. run `blocklet deploy .blocklet/bundle --app-did={your-app-did} --mount-point=/` and then try to start your app
4. ensure the running app works as expected
5. then we are ready to 

please help to ensure the above checklist before we can merge and release.

**Comment by vmcodes** [comment]
@wangshijun Thank you, I updated some of the code and had no issues deploying the template. Please let me know if anything needs to be changed.

nestjs-template-test

File: packages/create-app/templates/nestjs-api/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+APP_TITLE=''
```
Comment (LancelotLewis) [inline packages/create-app/templates/nestjs-api/.env:1]: If you don't use this env, you can remove this file

File: packages/create-app/templates/nestjs-api/README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+## File Structure
+
+- screenshots/ - Screenshots
+- api/ - Api side code
+  - hooks/ - blocklet lifecycle hooks
+  - libs/ - Api side libraries
+  - middlewares/ - Api side middlewares
+  - routes/ - Api side routes
+  - index.js - Api side entry point
```
Comment (LancelotLewis) [inline packages/create-app/templates/nestjs-api/README.md:15]: Please change this folder structure as this template is

**Review by LancelotLewis — CHANGES_REQUESTED** [review]
Please add this code in `/packages/create-app/index.js` L121
you can see `[API] nestjs` when use `blocklet create` command

``` js
{
  name: 'nestjs-api',
  display: '[api] nestjs',
  color: yellow,
},
```

**Comment by vmcodes** [comment]
@LancelotLewis thanks for your patience. The new updates should be correct. Let me know.

**Review by LancelotLewis — APPROVED** [review]

**Pull request merged**